tests/streamlit_ui_tests/toolbar/home.spec.ts

```diff
@@ -85,10 +85,11 @@ test.describe('Home Tab Buttons', () => {
   test('Delete Column', async ({ page }) => {
     const mito = await getMitoFrameWithTestCSV(page);
 
-    await mito.getByTitle('Column1').click();
+    await mito.getByTitle('Column2').click();
```

Comment:
Why does moving it from Column1 to Column2 fix the test?

Reply:
Honestly I don't know why he changed this hahah